src/app/multi-editor/multi-editor.component.ts

```diff
@@ -117,6 +123,7 @@ export class MultiEditorComponent implements OnInit {
       .then((json) => {
         this.previewMode = false;
         this.errorMessage = undefined;
+        this.successMessage = undefined;
```

Comment:
two lines above should outside of then or even in the method which is calling this.